vendor/vendor.json

```diff
@@ -3,6 +3,18 @@
 	"ignore": "test",
 	"package": [
 		{
+			"path": "appengine",
+			"revision": ""
```

Comment:
Those were internal deps for appengine.
All those without revision were guarded by an appengine only build tag.
I added appengine to the ignore tags of govendor which fixed those ones.
There was only one left which was external and accidentally not moved to the vendor folder. Fixed that as well.